topsf/config/run3/config_sf.py

```diff
@@ -0,0 +1,1414 @@
+                "lumi_13TeV_correlated": 0.006j,
+            })
+    else:
+        raise NotImplementedError(f"Luminosity for year {year} is not defined.")
+
+    #
+    # cross sections
+    #
+
+    # cross sections for diboson samples; taken from:
+    # - ww (NNLO): https://arxiv.org/abs/1408.5243
+    # - wz (NLO): https://arxiv.org/abs/1105.0020
+    # - zz (NNLO): https://www.sciencedirect.com/science/article/pii/S0370269314004614?via%3Dihub
+    diboson_xsecs_13 = {
+        "ww": Number(118.7, {"scale": (0.025j, 0.022j)}),
+        "wz": Number(46.74, {"scale": (0.041j, 0.033j)}),
+        # "wz": Number(28.55, {"scale": (0.041j, 0.032j)}) + Number(18.19, {"scale": (0.041j, 0.033j)}),  # (W+Z) + (W-Z)  # noqa
+        "zz": Number(16.99, {"scale": (0.032j, 0.024j)}),
+    }
+    # TODO Use 14 TeV xs for Run 3?
+    diboson_xsecs_14 = {
+        "ww": Number(131.1, {"scale": (0.026j, 0.022j)}),
+        "wz": Number(67.06, {"scale": (0.039j, 0.031j)}),
+        # "wz": Number(31.50, {"scale": (0.039j, 0.030j)}) + Number(20.32, {"scale": (0.039j, 0.031j)}),  # (W+Z) + (W-Z)  # noqa
+        "zz": Number(18.77, {"scale": (0.032j, 0.024j)}),
+    }
+
+    # linear interpolation between 13 and 14 TeV
+    diboson_xsecs_13_6 = {
+        ds: diboson_xsecs_13[ds] + (13.6 - 13.0) * (diboson_xsecs_14[ds] - diboson_xsecs_13[ds]) / (14.0 - 13.0)
+        for ds in diboson_xsecs_13.keys()  # ww: 125.8 wz: 58.932 zz: 18.058  noqa
+    }
+
+    for ds in diboson_xsecs_14:
+        procs.n(ds).set_xsec(13.6, diboson_xsecs_13_6[ds])
+
+    #
+    # pileup
+    #
+
+    # # minimum bias cross section in mb (milli) for creating PU weights, values from
+    # # https://twiki.cern.ch/twiki/bin/view/CMS/PileupJSONFileforData?rev=45#Recommended_cross_section
+    # cfg.x.minbias_xs = Number(69.2, 0.046j)
+
+    #
+    # MET filters
+    #
+
+    # https://twiki.cern.ch/twiki/bin/viewauth/CMS/MissingETOptionalFiltersRun2#Run_3_recommendations
+    cfg.x.met_filters = {
+        "Flag.goodVertices",
+        "Flag.globalSuperTightHalo2016Filter",
+        "Flag.EcalDeadCellTriggerPrimitiveFilter",
+        "Flag.BadPFMuonFilter",
+        "Flag.BadPFMuonDzFilter",
+        "Flag.eeBadScFilter",
+        "Flag.ecalBadCalibFilter",
+    }
+
+    #
+    # JEC & JER  # FIXME: Taken from HBW
+    # https://github.com/uhh-cms/hh2bbww/blob/master/hbw/config/config_run2.py#L138C5-L269C1
+    #
+
+    # jec configuration
+    # https://twiki.cern.ch/twiki/bin/view/CMS/JECDataMC?rev=2017#Jet_Energy_Corrections_in_Run2
+
+    # jec configuration taken from HBW
+    # https://github.com/uhh-cms/hh2bbww/blob/master/hbw/config/config_run2.py#L138C5-L269C1
+    # https://twiki.cern.ch/twiki/bin/view/CMS/JECDataMC?rev=201
+    jerc_postfix = ""
+    if year == 2022 and campaign.x.EE == "post":
+        jerc_postfix = "EE"
+
+    if cfg.x.run == 3:
+        jerc_campaign = f"Summer{year2}{jerc_postfix}_22Sep2023"
+        jet_type = "AK4PFPuppi"
```

Comment:
Here the check for run 3 can be skipped since this is already run 3 config.
```suggestion
    jerc_campaign = f"Summer{year2}{jerc_postfix}_22Sep2023"
    jet_type = "AK4PFPuppi"
```